<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ADDSTUDENTWINDOW</class>
 <widget class="QMainWindow" name="ADDSTUDENTWINDOW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>427</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>427</width>
    <height>180</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Edit Student Form</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="doneEdit">
    <property name="geometry">
     <rect>
      <x>314</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>381</width>
      <height>126</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="idLabel">
       <property name="text">
        <string>ID Number</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="idField">
       <property name="placeholderText">
        <string>e.g.&quot;2020-1971&quot;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Full Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="nameField">
       <property name="placeholderText">
        <string>e.g. &quot;Jason Dela Cruz</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="courseLabel">
       <property name="text">
        <string>Course</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QComboBox" name="courseField">
       <item>
        <property name="text">
         <string>BSCS - Computer Science</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BSIT - Information Techonology</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BSIS - Information Systems</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BSBA - Business Administration</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BSHRM - Hotel and Restaurant Management</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BSEE - Elementary Education</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BSCE - Computer Engineering</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BSME - Mechanical Engineering</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BAPS - Political Science</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BSEET - Environmental Engineering Technology</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="yearLabel">
       <property name="text">
        <string>Year Level</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="yearField">
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>20</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="genderLabel">
       <property name="text">
        <string>Gender</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="genderField">
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>20</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Male</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Female</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Others</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>idField</tabstop>
  <tabstop>nameField</tabstop>
  <tabstop>genderField</tabstop>
  <tabstop>doneEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
